components/Card.js

```diff
@@ -43,41 +71,57 @@ const Card = ({
     );
     return (
         <CardView>
-            <BigScrollStyledView
-                directionalLockEnabled={true}
-                contentContainerStyle={{ maxWidth: '99.9%' }}
-            >
-                <StandardTextbox text={category.toLowerCase()} color={color} />
-
-                <CardTextbox textList={textList} color={color} />
-                {hasFollowUp ? (
-                    <TwoButtonContainer>
+            <View style={{ height: '100%' }}>
+                <View style={{ height: '9%', justifyContent: 'stretch' }}>
+                    <CardTitle text={category.toUpperCase()} color={color} />
+                </View>
+                <View style={{ height: '1.5%' }}></View>
+                <View style={{ height: '77.5%' }}>
+                    <CardTextbox textList={textList} color={color} />
+                </View>
+                <View style={{ height: '1.5%' }}></View>
+                <View style={{ height: '9%' }}>
```

Comment:
Adding this in-line styling is a little inconsistent with the styled-component format that the rest of the project is using. Would be good if you make each component a styled-component (not a required change)

Reply:
Making a styled-component for just height differences seems like it would make the code less readable.

Reply:
It's probably fine for this one. This kind of thing is like adding `<br>` to a HTML code. Not ideal but it works.